styles/_dropdown.scss

```diff
@@ -0,0 +1,57 @@
+.dropdown {
+  position: relative;
+
+  &__options {
+    display: none;
+    position: absolute;
+    right: 0%;
+    bottom: 100%;
+    border: $border;
+
+    flex-direction: column;
+
+    background: $brand-white;
+
+    text-align: center;
+  }
+
+  &:hover {
+    .dropdown__options {
+      // both display: flex; & display: block; work for this assignment maybe?
+      // display: flex;
+      display: flex;
+    }
+  }
+}
+
+.dropdown__options-item {
```

Comment:
Should be able to use SCSS nesting here to make it clear that `__options-item` elements belong to the `dropdown` block.